packages/eslint-config-angular/index.js

```diff
@@ -1,14 +1,13 @@
 module.exports = {
     extends: [
         'plugin:@angular-eslint/recommended',
-        'plugin:@angular-eslint/recommended--extra',
```

Comment:
Do we know why recommended--extra was removed?

Reply:
No, [the release notes](https://github.com/angular-eslint/angular-eslint/blob/main/CHANGELOG.md#breaking-changes-1) just say it was removed (along with a couple others). Couldn't find a commit or issue that explained further.

Reply:
Thanks for looking.